src/raftstore/store/worker/compact.rs

```diff
@@ -123,6 +125,13 @@ impl Runner {
             1, /* threads */
         );
         compact_range_timer.observe_duration();
```

Comment:
How about using `observe`?

```rust
let timer = Instant::now();
let duration = timer.elapsed();
COMPACT_RANGE_CF.with_label_values(&[&cf_name]).observe(duration_to_sec(duration));
```

Reply:
@overvenus seem we need to provide a function to hide `duration_to_sec` later, not let the user do.